modules/map/tools/map_xysl.cc

```diff
@@ -140,51 +143,85 @@ class MapUtil {
     return 0;
   }
 
+  void PrintObjectOverlapInfo(const auto &object_info) {
+    const auto &raw_id = object_info.id().id();
+    switch (object_info.overlap_info_case()) {
+      case ObjectOverlapInfo::kLaneOverlapInfo: {
+        std::cout << "Lane : " << raw_id << std::endl;
+        PrintLane(GetLane(raw_id));
+        break;
+      }
+      case ObjectOverlapInfo::kSignalOverlapInfo: {
+        std::cout << "Signal: " << raw_id << std::endl;
+        Print(GetSignal(raw_id)->inner_object());
+        break;
+      }
+      case ObjectOverlapInfo::kStopSignOverlapInfo: {
+        std::cout << "StopSign: " << raw_id << std::endl;
+        Print(GetStopSign(raw_id)->inner_object());
+        break;
+      }
+      case ObjectOverlapInfo::kCrosswalkOverlapInfo: {
+        std::cout << "Crosswalk: " << raw_id << std::endl;
+        Print(GetCrosswalk(raw_id)->inner_object());
+        break;
+      }
+      case ObjectOverlapInfo::kJunctionOverlapInfo: {
+        std::cout << "Junction: " << raw_id << std::endl;
+        Print(GetJunction(raw_id)->inner_object());
+        break;
+      }
+      case ObjectOverlapInfo::kYieldSignOverlapInfo: {
+        std::cout << "YieldSign: " << raw_id << std::endl;
+        Print(GetYieldSign(raw_id)->inner_object());
+        break;
+      }
+      case ObjectOverlapInfo::kClearAreaOverlapInfo: {
+        std::cout << "ClearArea: " << raw_id << std::endl;
+        Print(GetClearArea(raw_id)->inner_object());
+        break;
+      }
+      case ObjectOverlapInfo::kSpeedBumpOverlapInfo: {
+        std::cout << "SpeedBump: " << raw_id << std::endl;
+        Print(GetSpeedBump(raw_id)->inner_object());
+        break;
+      }
+      case ObjectOverlapInfo::kParkingSpaceOverlapInfo: {
+        std::cout << "ParkingSpace: " << raw_id << std::endl;
+        break;
+      }
+      case ObjectOverlapInfo::kPncJunctionOverlapInfo: {
+        std::cout << "PncJunction: " << raw_id << std::endl;
+        Print(GetPNCJunction(raw_id)->inner_object());
+        break;
+      }
+      // case ObjectOverlapInfo::kRsuOverlapInfo: {
+      //   std::cout << "RSU: " << raw_id << std::endl;
+      //   Print(GetRSU(raw_id)->inner_object());
+      //   break;
+      // }
```

Comment:
Definition of GetRSU was not found leading to linking error. So currently is commented.